### DISCUSSION TITLE
Not able to install the packages inside react native example app.

### DISCUSSION THREAD

**Discussion by archana-iron** [callstack/react-native-builder-bob#556] [Q&A]
### Description

Please check this once: https://github.com/facebook/react-native/issues/44611

### Packages

- [X] create-react-native-library
- [X] react-native-builder-bob

### Selected options

I have created react-native custom library using 'create-react-native-library' but this library example is not working. Because I am not able to install the packages inside my example library. Also I am suing npm instead of using yarn.

### Link to repro

https://github.com/facebook/react-native/issues/44611

### Environment

System:
  OS: macOS 13.0
  CPU: (10) arm64 Apple M1 Max
  Memory: 151.39 MB / 32.00 GB
  Shell:
    version: 5.8.1
    path: /bin/zsh
Binaries:
  Node:
    version: 21.7.3
    path: /opt/homebrew/bin/node
  Yarn:
    version: 1.22.22
    path: /opt/homebrew/bin/yarn
  npm:
    version: 10.5.0
    path: /opt/homebrew/bin/npm
  Watchman:
    version: 2024.04.22.00
    path: /opt/homebrew/bin/watchman
Managers:
  CocoaPods:
    version: 1.15.2
    path: /Users/archanasharma/.rvm/gems/ruby-2.7.6/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 22.2
      - iOS 16.2
      - macOS 13.1
      - tvOS 16.1
      - watchOS 9.1
  Android SDK: Not Found
IDEs:
  Android Studio: 2023.2 AI-232.10300.40.2321.11668458
  Xcode:
    version: 14.2/14C18
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 17.0.11
    path: /usr/bin/javac
  Ruby:
    version: 2.7.6
    path: /Users/archanasharma/.rvm/rubies/ruby-2.7.6/bin/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react: Not Found
  react-native: Not Found
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found

(node:12187) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)

**Comment by satya164** [comment]
https://callstack.github.io/react-native-builder-bob/faq#how-to-test-the-library-in-an-app-locally
